### PR TITLE
New Post: Java Resource Packs

### PR DESCRIPTION
**Pull request by KuroiJigoku**

### CODE REVIEW

File: _posts/minecraft/java/2024-07-13-java-rp.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Resource Packs"
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-13-java-rp.md:3]:
```diff
- title:  "Resource Packs"
+ title: "How to add a resource pack to your server"
```

File: _posts/minecraft/java/2024-07-13-java-rp.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+category: 
+    - Java
+    - Getting-started
+tags: Server
+description: "Uploading Resource Packs into your Java Server"
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-13-java-rp.md:8]:
```diff
- description: "Uploading Resource Packs into your Java Server"
+ description: "How to make your Java server require a Resource Pack"
```

File: _posts/minecraft/java/2024-07-13-java-rp.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- [Minecraft Forum](https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/resource-packs "A longstanding community forum where players discuss and share Minecraft-related content, including resource packs.")
+- [Modrinth](https://modrinth.com/resourcepacks "Modrinth is a platform tailored for Minecraft players and mod developers, offering a curated selection of mods, texture packs, and community content.")
+
+You can also upload your own custom resource pack as well.
+
+{: .warning}
+
+> **Compatibility Issues**: Ensure the resource pack is compatible with the version of Minecraft Java Edition and any server plugins or mods installed. Incompatible packs may cause crashes or visual glitches.
+
+## How to Add a Resource Pack to the Minecraft Server
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-13-java-rp.md:30]:
```diff
- ## How to Add a Resource Pack to the Minecraft Server
+ ## Adding a Resource Pack to the server
```

File: _posts/minecraft/java/2024-07-13-java-rp.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+1. Upload the resource pack to [Dropbox](https://www.dropbox.com/home) or a similar filesharing service.
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-13-java-rp.md:32]: Dropbox 🙄

File: _posts/minecraft/java/2024-07-13-java-rp.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+4. Go to the [File Manager](https://client.falixnodes.net/server/filemanager?dir=/) in the Client Panel.
+
+5. Find the `server.properties` file and open it.
+
+6. Within the file, find the attribute `resource-pack=` and paste the previously copied link as the attribute's value.
+
+7. Save the file
+
+8. (re)Start the server
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-13-java-rp.md:52]:
```diff
- 8. (re)Start the server
+ 8. Start or restart the server
```

**Review by iamdeka — CHANGES_REQUESTED** [review]

File: _posts/minecraft/java/2024-07-13-java-rp.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    - Getting-started
+tags: Server
+description: "Uploading Resource Packs into your Java Server"
+permalink: /minecraft/java/server/resource-pack
+author: "Kuroi Jigoku"
+toc: false
+---
+## Introduction
+
+Resource packs are a popular way to customize the visual and auditory aspects of Minecraft Java Edition. Whether you're looking to enhance the game with realistic textures, add thematic elements, or completely overhaul the look and feel, Java Edition resource packs offer endless possibilities. This guide explores where to find and how to install these packs, enabling you to personalize your Minecraft experience to suit your preferences and style.
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-13-java-rp.md:15]: I smell the AI

File: _posts/minecraft/java/2024-07-13-java-rp.md (hunk 4 above)
Comment (KuroiJigoku) [reply _posts/minecraft/java/2024-07-13-java-rp.md:32]: men...its the easy one out there 🙄

**Review by Mocab — APPROVED** [review]
If anyone has anything else to add lmk

**Comment by KuroiJigoku** [comment]
Deka approved :D
me got nothin' else to edit on i think :3

**Pull request merged**